### ISSUE TITLE
[PROPOSAL] Abnormal Titans turning more slowly 

### ISSUE THREAD

**Issue by Alex-Mod-Bot152** [AoTTG-2/AoTTG-2#401]
Idea by @Emperor Stiffy
#8301 

Idea link-https://discord.com/channels/681641241125060652/683376807101071489/790472006448840705

**How will this enhancement improve the game?**
Making Abnormals titans feel more real by adding "weight" to them. Nothing with great mass comes to full stop in very short time

**Describe the solution you'd like**
Making the turn more slowly, possibly by even in drift motion

**Describe alternatives you've considered**
Abnormals slowing down before they start turning 

**Contributor instructions** (Do not modify)
1. Determine if the enhancement is compatible with existing features
2. If the idea is denied, please provide a reason and close the issue. If the idea is accepted, consult your team to do an estimation of effort. Remove the [PROPOSAL] name and label.
3. Move the issue to the Future Milestone. On rare exceptions an issue may be moved to Release if there's sufficient upvotes.
4. Once the issue matches the current milestone, follow development procedures.

**Comment by Prolonpo** [comment]
Makes sense, this small addition should make the game more realistic and less weird with a giant mass of flesh stopping in a fraction of a second.
